core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/BillingAccountChangeSynchronizer.scala

```diff
@@ -202,6 +202,9 @@ final case class BillingAccountChangeSynchronizer(dataSource: SlickDataSource,
 
       outcome <- workspaces.foldMapA { workspace =>
         for {
+          _ <- inTransaction {
+            workspaceQuery.withWorkspaceId(workspace.workspaceIdAsUUID).setState(WorkspaceState.Updating)
```

Comment:
Have you been able to test if this is working correctly?

Reply:
I will check by running locally/on dev once Sam is back up, but if I just change the later code to never set it to `UpdateFailed`/`Ready`, the test does appear to confirm that it gets set to `Updating`.

Reply:
I was asking if you had test coverage for it to make sure this bit worked.

Reply:
No, I haven't figured out yet how to go about testing it directly.